frontend/src/domain/types/missionAction.ts

```diff
@@ -1,173 +1,183 @@
-  LAND_CONTROL = 'LAND_CONTROL',
-  OBSERVATION = 'OBSERVATION',
-  SEA_CONTROL = 'SEA_CONTROL'
-}
+  // ---------------------------------------------------------------------------
+  // Constants
 
-enum ControlCheck {
-  NO = 'NO',
-  NOT_APPLICABLE = 'NOT_APPLICABLE',
-  YES = 'YES'
-}
+  export enum ControlCheck {
+    NO = 'NO',
+    NOT_APPLICABLE = 'NOT_APPLICABLE',
+    YES = 'YES'
+  }
+  export enum InfractionDomain {
+    GEAR = 'GEAR',
+    LOGBOOK = 'LOGBOOK',
+    OTHER = 'OTHER',
+    SPECIES = 'SPECIES'
+  }
 
-export type Controller = {
-  administration: string
-  controller: string
-  controllerType: string
-}
+  /* eslint-disable typescript-sort-keys/string-enum */
+  export enum InfractionType {
+    WITH_RECORD = 'WITH_RECORD',
+    WITHOUT_RECORD = 'WITHOUT_RECORD',
+    PENDING = 'PENDING'
+  }
+  export const InfractionTypeLabel: Record<InfractionType, string> = {
+    [InfractionType.WITH_RECORD]: 'Avec PV',
+    [InfractionType.WITHOUT_RECORD]: 'Sans PV',
+    [InfractionType.PENDING]: 'En attente'
+  }
+  /* eslint-enable typescript-sort-keys/string-enum */
+
+  export enum MissionActionType {
+    AIR_CONTROL = 'AIR_CONTROL',
+    AIR_SURVEILLANCE = 'AIR_SURVEILLANCE',
+    LAND_CONTROL = 'LAND_CONTROL',
+    OBSERVATION = 'OBSERVATION',
+    SEA_CONTROL = 'SEA_CONTROL'
+  }
 
-export enum InfractionType {
-  PENDING = 'PENDING',
-  WITHOUT_RECORD = 'WITHOUT_RECORD',
-  WITH_RECORD = 'WITH_RECORD'
-}
+  // ---------------------------------------------------------------------------
+  // Helpers
 
-export enum InfractionDomain {
-  GEAR = 'GEAR',
-  LOGBOOK = 'LOGBOOK',
-  OTHER = 'OTHER',
-  SPECIES = 'SPECIES'
-}
+  export const isGearInfraction = (p: any): p is GearInfraction => p.gearSeized !== undefined
+  export const isSpeciesInfraction = (p: any): p is SpeciesInfraction => p.speciesSeized !== undefined
 
-export const isGearInfraction = (p: any): p is GearInfraction => p.gearSeized !== undefined
-export const isSpeciesInfraction = (p: any): p is SpeciesInfraction => p.speciesSeized !== undefined
+  // ---------------------------------------------------------------------------
+  // Types
 
-export type GearInfraction = {
-  comments: string
-  gearSeized: boolean
-  infractionType: InfractionType
-  natinf: number
-}
+  export type ControlAndText = {
+    control: MissionAction | undefined
+    text: string
+  }
 
-export type SpeciesInfraction = {
-  comments: string
-  infractionType: InfractionType
-  natinf: number
-  speciesSeized: boolean
-}
+  export type Controller = {
+    administration: string
+    controller: string
+    controllerType: string
+  }
 
-export type LogbookInfraction = {
-  comments: string
-  infractionType: InfractionType
-  natinf: number
-}
+  export type FleetSegment = {
+    segment: string | undefined
+    segmentName: string | undefined
 
-export type OtherInfraction = {
-  comments: string
-  infractionType: InfractionType
-  natinf: number
-}
+    // TODO I had to add that.
```

Comment:
Why this TODO ?

Reply:
Because it's not in the existing types: https://github.com/MTES-MCT/monitorfish/blob/master/frontend/src/domain/types/missionAction.ts#L87